diesel/src/types/mod.rs

```diff
@@ -380,6 +421,7 @@ impl<T: NotNull + SingleValue> SingleValue for Nullable<T> {}
 ///
 /// [`MysqlType`]: ../mysql/enum.MysqlType.html
```

Comment:
We should mention somewhere here that all types that implement `FromSql` also must also implement `FromSqlRow` (and link to `FromSqlRow`)